components/cart.html

```diff
@@ -43,6 +43,11 @@
 						<i class="far fa-shopping-cart fa-lg"></i>
 					</div>
 				</a>
+				<a href="/"
```

Comment:
You might want to format your code a bit.

Reply:
yeah i know , for some reason my prettier isn't working well . causes problem in only anchor tags.